src/foam/u2/wizard/wao/PrerequisiteWAO.js

```diff
@@ -15,8 +15,7 @@
   `,
 
   properties: [
-    foam.u2.wizard.data.PrerequisiteLoader
-      .getOwnAxiomsByClass(foam.core.Property).map(a => a.clone()),
+    ...(k => foam.USED[k] || foam.UNUSED[k])('foam.u2.wizard.data.PrerequisiteLoader').properties,
```

Comment:
Why do this this way?

Reply:
@KernelDeimos my bad didn't have the solution we figured out afterwards in the code. We can change this spread to     ```
...foam.u2.wizard.data.PrerequisiteLoader
      .getOwnAxiomsByClass(foam.core.Property).map(a => a.clone()),
```

Reply:
@kgrgreer @tharmaman updated by copying the properties here manually. A better solution is [provided here](https://github.com/kgrgreer/foam3/pull/1654).